server/statistics/hot_peer.go

```diff
@@ -192,3 +191,10 @@ func (stat *HotPeerStat) clearLastAverage() {
 		l.clearLastAverage()
 	}
 }
+
+func (stat *HotPeerStat) expectedInterval() int {
```

Comment:
```suggestion
func (stat *HotPeerStat) otStatReportInterval() int {
```
how about `hotStatReportInterval`?